### DISCUSSION TITLE
does not display connections

### DISCUSSION THREAD

**Discussion by Under0Cover** [damms005/devdb-vscode#50] [General]
Follow the instructions to get started.
At the end of the settings, the extension showed both the option to connect via .devdbrc and integration with Laravel.
However, it currently no longer displays connections. Neither made by .devdbrc nor by Laravel

![image](https://github.com/damms005/devdb-vscode/assets/60072961/314d88ff-ec8f-4bf5-9b2a-3415b2237afe)

**Comment by damms005** [comment]
Please [open an issue](https://github.com/damms005/devdb-vscode/issues/new?assignees=&labels=bug%2Cunconfirmed%2Clow+priority&projects=&template=bug_report.yml) so that you can provide appropriate information needed to triage this.

Thank you!
